ContaPessoas/Contador.m

```diff
@@ -39,7 +52,16 @@ -(int)getGirls {
     return girl;
 }
 
+-(int)getTotal {
+    return boy + girl;
+}
+
 
+-(void)atualiza{
+    if (self.delegate && [self.delegate respondsToSelector:@selector(atualiza)]) {
```

Comment:
Se a instancia ja implementa o protocolo, nao deveria ser necessário fazer o responds to.